docs/references/nextjs/auth.mdx

```diff
@@ -11,7 +11,40 @@ The `auth()` helper does require [Middleware](/docs/references/nextjs/auth-middl
 
 A Route Handler added to [`publicRoutes`](/docs/references/nextjs/auth-middleware#making-pages-public-using-public-routes) can still use the [`auth()`](/docs/references/nextjs/auth) helper to return information about the user or their authentication state, or to control access to some or all of the Route Handler.
 
-## Retrieving userId
+## Return type of `auth()`
+
+`auth()` returns the [`Authentication`](/docs/references/nextjs/authentication-object) object with a few extra properties:
+
+| Name | Type | Description |
```

Comment:
Since we fully explain these properties in the H3s below, what do you think of turning this table into a two-item bulleted list that just links to the below sections? Less repetition = easier to maintain IMO